tests/pcmsolver/ccsd-pte/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+include(TestingMacros)
+
+add_regression_test(pcmsolver-ccsd-pte "psi;quicktests;pcmsolver;addon;ccsd;pte")
```

Comment:
I use the labels quite extensively for testing. If we have them it should be consistent IMO. A new CCSD tag here would mean that we would need to tag all cases that are CCSD only.